setup.py

```diff
@@ -322,7 +325,7 @@ def build_oneapi_backend():
         includes=includes,
         eca=eca,
         ela=ela,
-        defines=[],
+        defines=[onedal_version_define],
```

Comment:
This define breaks consistency of version check in all python wrappers where version is received from version header

Reply:
Any other proposals?

Reply:
All these tweaks with versions are required for green CI just because 2023.1 have not been published yet. I will remove all this stuff ASAP after the release

Reply:
Fixed